cli/jobs/pipelines-with-components/basics/4d_dataset_input/pipeline.yml

```diff
@@ -1,11 +1,12 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
 type: pipeline
+display_name: 4d_dataset_input
 description: "Example of using data from a Dataset as pipeline input"
 inputs:
   pipeline_sample_input_data:
     type: mltable
     path: azureml:sampledata@latest
-    mode: eval_mount
```

Comment:
@wangchao1230 note that eval_mount is not a valid mode in sdk for now

Reply:
in cli?

Reply:
I think both sdk & cli? It will fail in schema check stage

Reply:
It will work in new sdk/cli version